docs/page.rst

```diff
@@ -625,6 +626,67 @@ In a nutshell, this is what you can do with PyMuPDF:
+      pair: text; insert_htmlbox
+      pair: css; insert_htmlbox
+      pair: adjust; insert_htmlbox
+      pair: archive; insert_htmlbox
+      pair: overlay; insert_htmlbox
+      pair: rotate; insert_htmlbox
+      pair: oc; insert_htmlbox
+      pair: morph; insert_htmlbox
+
+   .. method:: insert_htmlbox(rect, text, *, css=None, scale_low=0, archive=None, rotate=0, oc=0, overlay=True)
+
+      * New in v1.23.8
+
+      PDF only. Insert text into the specified rectangle. The method has similarities with methods :meth:`Page.insert_textbox` and :meth:`TextWriter.fill_textbox`, but is **much more powerful**. This is achieved by letting a :ref:`Story` object do all the required processing.
+
+      * Parameter "text" may be a string as in the other methods. But it will be **interpreted as HTML source** and may therefore also contain HTML language elements -- including styling. The "css" parameter may be used to pass in additional styling instructions.
+
+      * Automatic line breaks are inserted at word boundaries. The "soft hyphen" character `"&#173;"` can be used to cause hyphenation and thus also cause line breaks. **Forced** line breaks however are only achievable via the HTML tag `<br>` - `"\\n"` is ignored and will be treated like a space.
+
+      * With this method the following can be achieved:
+
+         - Styling effects like bold, italic, text color, text alignment, font size or font switching.
+         - The text may inlude arbitrary languages -- **including rigt-to-left** languages.
+         - Scripts like `Devanagari <https://en.wikipedia.org/wiki/Devanagari>`_ and several others in Asia have a highly complex system of ligatures, where two or more unicodes together yield one glyph. The Story uses the software package `HarfBuzz <https://harfbuzz.github.io/>`_ , to deal with these things and produce correct output.
+         - One can also **include images** via HTML tag `<img>` -- the Story code will take care of the appropriate layout. This is an alternative option to insert images, compared to :meth:`Page.insert_image`.
+         - HTML tables (tag `<table>`) may be included in the text and will be handled appropriately.
+         - Links are automatically generated when present.
+
+      * If content does not fit in the rectangle, the developer has two choices:
+         
+         - **either** be just informed (and accept a no-op), 
+         - **or** (`scale=True` - the default) scale down the content until it fits.
+
+      :arg rect_like rect: rectangle on page to receive the text.
+      :arg str,Story text: the text to be written. Can contain plain text and HTML tags with styling instructions. Alternatively, a :ref:`Story` object may be specified (in which case the internal Story generation step will be omitted). A Story must have been generated with all required styling and Archive information.
+      :arg str css: optional string containing additional CSS instructions. Ignored if "text" is a Story.
```

Comment:
Again if you decide parameter "text" is better as `text` then we would need to do this throughout :)
I'm not sure now ....